participants/IPFS-Found.md

```diff
@@ -0,0 +1,61 @@
+# <Project Name>
```

Comment:
This application has not been filled out.

Please answer the questions within the document to qualify.